### DISCUSSION TITLE
is:issue is:open docker pull fail,manifest for docker.io/starrocks/dev-env-centos7:latest not found,please help

### DISCUSSION THREAD

**Discussion by samfeng2003** [StarRocks/starrocks#36021] [How-tos, Tutorials, Best Practices and Architecture Articles]
why pull fail?
[root@99 ~]# docker pull starrocks/dev-env-centos7:latest
Trying to pull repository docker.io/starrocks/dev-env-centos7 ... 
latest: Pulling from docker.io/starrocks/dev-env-centos7
manifest for docker.io/starrocks/dev-env-centos7:latest not found

please help.

**Comment by alberttwong** [comment]
this isn't the place to ask questions.
